code/modules/surgery/organs/robolimbs.dm

```diff
@@ -11,12 +12,15 @@ var/global/datum/robolimb/basic_robolimb
 			if(R != "head" && R != "chest" && R != "groin" ) //Part of the method that ensures only IPCs can access head, chest and groin prosthetics.
 				if(R.has_subtypes) //Ensures solos get added to the list as well be incorporating has_subtypes == 1 and has_subtypes == 2.
 					chargen_robolimbs[R.company] = R //List only main brands and solo parts.
+		if(R.selectable)
```

Comment:
This proc has absolutely nothing to do with that problem.

I'm unable to replicate the problem mentioned in the issue report, locally, but I have an idea of what is causing it.

There's no way at all this proc is the problem.